getline.ts/src/example.ts

```diff
@@ -13,6 +13,10 @@ let main = async() => {
         let loan = loans[i];
         await loan.updateStateFromBlockchain();
         console.log("Loan " + loan.shortId);
+        let amountWanted = await loan.parameters.loanToken.humanize(loan.parameters.amountWanted);
+        let symbol = await loan.parameters.loanToken.symbol();
+        console.log("  amount wanted:        " + amountWanted.toString() + " " + symbol);
```

Comment:
We can use deconstructing:
```
const {loanToken, interestPermil, paybackDeadline, paybackDeadline} = loan.parametrs;
```
instead of
``` 
loan.parametrs.loanToken, loan.parametrs.interestPermil...
```

Reply:
Done.